tasks/install.debian.yml

```diff
@@ -9,7 +9,7 @@
 - name: Establish some role-related facts
   set_fact:
     mongodb_is_systemd: "{{ sbin_init.stat.islnk is defined and sbin_init.stat.islnk }}"
-    mongodb_major_version: "{{ mongodb_version[0:3] }}"
+    mongodb_major_version: "{{ mongodb_version[0:4] }}"
```

Comment:
Why did you change that slice here? So you changed, for example: `3.6.2` from `3.6` to `3.6.`

Reply:
This change was done by @ftx, but I think you are right. I'll change it back to `[0:3]`, unless @ftx gives a reason for this change.

Reply:
Reverted this change.